scripts/apishow.py

```diff
@@ -232,16 +233,21 @@ def _print_resource(res_title, res_dict):
         help="path to cfme log file, default: %(default)s")
     args = parser.parse_args()
 
+    if not args.address:
+        address = get_or_create_current_appliance().address
+    else:
+        address = args.address
+
     # we are really not interested in any warnings and "warnings.simplefilter('ignore')"
     # doesn't work when it's redefined later in the REST API client
     warnings.showwarning = lambda *args, **kwargs: None
 
     api = MiqApi(
-        '{}/api'.format(args.address.rstrip('/')),
+        '{}/api'.format(address.rstrip('/')),
```

Comment:
**Optional**
Consider using os.path.join here to avoid potential for multiple slashes